### PR TITLE
Style table view input rows and remove rows.

### PR DESCRIPTION
**Pull request by gauthierm**
https://www.pivotaltracker.com/story/show/89167926

### CODE REVIEW

File: www/styles/admin-swat-local.css [hunk 1]
```diff
@@ -673,6 +673,31 @@ input.swat-entry.swat-insensitive {
 }
 
 
+/* Textarea Editor */
```
Comment (gauthierm) [inline www/styles/admin-swat-local.css]: These changes are supposed to be in another PR!

**Comment by sgarrity** [comment]
We could probably start using an `at2x` mixin for retina images (or SVG images in the admins), but no need to block on this.
